tools/autograd/templates/VariableType.cpp

```diff
@@ -38,10 +38,14 @@ using namespace torch::autograd::generated;
 namespace torch { namespace autograd {
 // Helper methods for working with Attributes (torch/csrc/jit/attributes.h)
 
+at::Tensor maybeUnwrapVar(const at::Tensor& t) {
```

Comment:
This seems a bit awkward. What was the proximal cause for this?

Reply:
This fixes #8755. The problem was that once we call `.toTensor()` on `at::Scalar`s below then we usually get a non-variable tensor, but we recently made a switch to use variables everywhere.